robot.js

```diff
@@ -94,70 +115,126 @@
     let x = robot.x;
     let y = robot.y;
 
+    /*changing these ternary operators to if/else statements: 
```

Comment:
This is now old, commented out code that should be removed when submitting a pull request (of course, the comments relating questions can stay because that is part of the assignment).